### ISSUE TITLE
Selectfield cut off by mdl card

### ISSUE THREAD

**Issue by gregor160300** [mebibou/mdl-selectfield#32]
Hello, I have been happily using this addon for mdl for a while now, but there is one super annoying bug that I can't seem to fix with CSS.

![mdl-selectfield cut off](https://cloud.githubusercontent.com/assets/7871398/21471621/14d7edd8-cab8-11e6-84f4-f6fa74927ace.png)
As you can see in the screenshot the dropdown is cut of and I have tried fixing it with z-index, overflow and other CSS but nothing seems to work.

As far as I can see there are 2 options here.
1. Make the dropdown menu goes up instead off down
2. Make the dropdown stay on top

I hope anyone can help me to implement either 1 of the 2 suggested fixes.

P.S. please don't suggest putting the dropdown at the top of the mdl-card to avoid the issue

**Comment by mebibou** [comment]
This is not an issue with the selectfield plugin, but with the card obviously. I think anything in the body of the card can not overflow, correct? So I will close this issue, but I would suggest you search on stackoverflow for overflow issue on mdl card or post a question there, as this is an implementation issue and not a plugin design issue.

**Comment by gregor160300** [comment]
Thank you for the tip and you were right. I would like to leave the fix here for others with the same problem.

I found this https://github.com/MEYVN-digital/mdl-selectfield/issues/2
it didn't completely fix the issue since it just cut off at the bottom of the entire card so I had to make it this
`
 .mdl-card__supporting-text, .mdl-card {
    overflow: visible;
}
`

I however have not tested if this breaks anything else.
